docs/blog/newsletter-may-2020.rst

```diff
@@ -0,0 +1,91 @@
+.. post:: May 04, 2020
+   :tags: newsletter
+
+####################################
+Write the Docs Newsletter – May 2020
+####################################
+
+Hello, documentarians; it's Beth here with the May edition of the Write the Docs newsletter. I hope you're all holding up okay under lockdown.
+
+We just announced last week that we're cancelling our on-site conferences in Portland and Prague, and moving online. Virtual Portland will be in August, with exact dates to be announced soon; and virtual Prague will be in the autman, with dates announced in summer. There's a lot more detail in`the blog post </blog/moving-portland-prague-2020-online/>`__, including what to do if you've already got a Portland ticket.
+
+To help you entertain yourself between now and August, there's some good news from the Write the Docs website. We've got an extensive archive of conference talks and newsletter articles, covering all sorts of topics, but up until now it's been a bit difficult to browse them all. So I'm delighted to introduce the the `WTD content index </topics/>`__. If there's a topic you'd like to spend some time learning about, check out the index and see if your topic is there!
+
+Onto our articles :)
+
+------------------------------------------------
+The pros and cons of FAQs
+------------------------------------------------
+
+
+---------------------------
+Getting more info from SMEs
+---------------------------
+
+No documentarian is an island: interacting with subject matter experts (SMEs) is often crucial to getting our jobs done. But what if the information these SMEs provide is too brief or incomplete? Here are some ideas on how to gather detailed and helpful information from these key team members:
+
+* Remember to have empathy. If SMEs don’t provide the answers you want, it doesn’t mean they don’t care. They’re busy people who are intimately familiar with their work; short answers make perfect sense to them as they already have a strong background in the topic.
```

Comment:
"intimately familiar with their work" feels a little obvious. How about "busy people with their own priorities"? Still obvious...

Reply:
and how about grounding instead of back ground?

Reply:
Made edits to both, thanks!